spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSkipperStreamServiceTests.java

```diff
@@ -139,20 +139,20 @@ public void createStreamWithMissingApps() {
 		thrown.expectMessage("Application name 'time' with type 'source' does not exist in the app registry.\n" +
 				"Application name 'log' with type 'sink' does not exist in the app registry.");
 
-		this.defaultSkipperStreamService.createStream("testStream", "time | log", false, false);
+		this.defaultSkipperStreamService.createStream("testStream", "time | log", false);
 	}
 
-	@Test
-	public void createStreamInvalidDsl() {
-		when(this.appRegistryCommon.appExist("time", ApplicationType.source)).thenReturn(true);
-		when(this.appRegistryCommon.appExist("log", ApplicationType.sink)).thenReturn(true);
-
-		thrown.expect(InvalidStreamDefinitionException.class);
-		thrown.expectMessage("Cannot determine application type for application 'koza': koza had " +
-				"neither input nor output set");
-
-		this.defaultSkipperStreamService.createStream("testStream", "koza", false, false);
-	}
+	//	@Test
```

Comment:
The test now fails with `Application name 'koza' with type 'app' does not exist in the app registry.` is ok, so updating to conform to that message.